### DISCUSSION TITLE
Expose Tuya scenes to Homekit

### DISCUSSION THREAD

**Discussion by Sebel1994** [milo526/homebridge-tuya-web#515] [Q&A]
Hello, when I import scenes into Homekit via the plugin, they are only displayed to me as devices (switches) and the scene cannot be controlled via Homekit.

Am I doing something wrong?

many thanks for the help

**Comment by milo526** [comment]
You can expose Tuya scenes as switches (as you found out); there is no way to expose them as a homekit scene.

Tuya scenes are not loaded by default; they are introduced to add a partial level of support for devices that otherwise could not be supported by this plugin.

As far as I am aware; nothing outside of Homekit itself can control scenes, even if this plugin could control scenes in HomeKit, there is no way to get data out of the scenes from Tuya.
